docs/fsharp/tools/fsharp-interactive/index.md

```diff
@@ -158,6 +158,16 @@ let f (x: Tensor) = sin (sqrt x)
 printfn $"{f (dsharp.tensor 1.2)}"
 ```
 
+By default ````#r "nuget: ...."```` will not use build targets from the package being referenced during restore.  There is a usepackagetargets option to enable the use of these build targets when required, only add usepackagetargets=true if the referenced package was authored to require it during restore.
+examples:
+
+```fsharp
```

Comment:
It would be good if we knew any popular(ish) package that indeed needs this and make an example out of it here, do we?

Reply:
I agree that it would be good to add meaningful examples here. I'll merge as is, and let this be a separate PR. 

/cc @T-Gro @KevinRansom